<?php

namespace ContainerErR77ez;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\ContainerInterface;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getMainAuthenticator2Service extends App_KernelDevDebugContainer
{
    /**
     * Gets the private 'debug.App\Security\MainAuthenticator' shared service.
     *
     * @return \Symfony\Component\Security\Http\Authenticator\Debug\TraceableAuthenticator
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 4).'/vendor/symfony/security-http/Authenticator/AuthenticatorInterface.php';
        include_once \dirname(__DIR__, 4).'/vendor/symfony/security-http/Authenticator/InteractiveAuthenticatorInterface.php';
        include_once \dirname(__DIR__, 4).'/vendor/symfony/security-http/EntryPoint/AuthenticationEntryPointInterface.php';
        include_once \dirname(__DIR__, 4).'/vendor/symfony/security-http/Authenticator/Debug/TraceableAuthenticator.php';

        return $container->privates['debug.App\\Security\\MainAuthenticator'] = new \Symfony\Component\Security\Http\Authenticator\Debug\TraceableAuthenticator(($container->privates['App\\Security\\MainAuthenticator'] ?? $container->load('getMainAuthenticatorService')));
    }
}
